### PR TITLE
Add net.line72.campcounselor

### PR DESCRIPTION
**Pull request by line72**
### Please confirm your submission meets all the criteria

- [X ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X ] My pull request follows the instructions at [App Submission][submission].
- [X ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: net.line72.campcounselor.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - /man
+  - /share/man
+  - /share/gtk-doc
+  - '*.la'
+  - '*.a'
+  - /lib/girepository-1.0
+  - /share/doc
+  - /share/gir-1.0
+modules:
+  - name: intltool
```
Comment (hfiguiere) [inline net.line72.campcounselor.yml:28]: please use the shared-modules.

https://github.com/flathub/shared-modules/

**Comment by hfiguiere** [comment]
bot, build net.line72.campcounselor

File: net.line72.campcounselor.yml (hunk 1 above)
Comment (line72) [reply net.line72.campcounselor.yml]: done.

**Comment by line72** [comment]
libgda 6 appears to be failing to build. It builds fine locally using flatpak-builder...

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/net.line72.campcounselor.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• file-invalid          : <screenshot> failed to load [https://line72.net/sn1.png]
• file-invalid          : <screenshot> failed to load [https://line72.net/sn2.png]
• file-invalid          : <screenshot> failed to load [https://line72.net/sn3.png]
• tag-missing           : <release> required
• tag-missing           : <url> is not present
Validation of files failed
```

that's the failure on x86_64.

**Comment by line72** [comment]
bot, build net.line72.campcounselor

**Comment by line72** [comment]
bot, build net.line72.campcounselor

**Comment by line72** [comment]
The aarch64 build is failing on libgda 6 which is a known issue:

https://gitlab.gnome.org/GNOME/libgda/-/issues/253

**Comment by hfiguiere** [comment]
this package includes a patch 

https://github.com/flathub/org.gnome.Gtranslator/blob/master/org.gnome.Gtranslator.yml#L58

**Comment by hfiguiere** [comment]
it's the one mentionned on the issue above.

**Comment by line72** [comment]
bot, build net.line72.campcounselor

**Comment by line72** [comment]
bot, build net.line72.campcounselor

**Comment by line72** [comment]
bot, build net.line72.campcounselor

**Comment by barthalion** [comment]
You should add --device=dri to finish-args. PTAL after the merge.

/merge

**Pull request closed without merging**